docs/source/notebooks/SMC2_gaussians.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>I don't understand why you're saying the ESS is 2 -- I see about 50 in the plot above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Good catcht .there are different ESSs. ESS bulk (the ess for the "central part" of the distribution is around 2, but the plots are showing local ESSs . I will clarify this.

Reply:
Ah ok! But how can local ESS be higher than ESS for the whole bulk? I would expect the opposite if bulk includes several local neighborhoods 🤔

Reply:
I added a few lines. I will improve the explanations in the ArviZ educational resources